src/pymor/models/interface.py

```diff
@@ -179,9 +259,9 @@ def _compute_output_error_estimate(self, solution, mu=None, **kwargs):
 
     _compute_allowed_kwargs = frozenset()
 
-    def compute(self, solution=False, output=False,
-                solution_error_estimate=False, output_error_estimate=False, *,
-                mu=None, **kwargs):
+    def compute(self, solution=False, output=False, solution_d_mu=False, output_d_mu=False,
+                solution_error_estimate=False, output_error_estimate=False,
+                return_output_d_mu_array=False, *, mu=None, **kwargs):
```

Comment:
`output_d_mu_return_array` would make much more sense to me (which is what I remember we settled on). Do you strongly prefer this name?

Reply:
hehe, I was just not able to remember the name within 10 minutes... Thanks for reminding me.